UEBPrep/card_list.csv

```diff
@@ -0,0 +1,15 @@
+cardtype,cardtitle,cardtext
```

Comment:
card type should be 'text' or 'rule'. If anything else or if blank, defaults to 'text'. A 'rule' card must have a card title. Rows can not be blank, will result in error.